cmd/routedns/dockerbuild.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+set -e
+
+arch=$(uname -m)
+
+if [ "$arch" = "x86_64" ]; then
```

Comment:
Hate to be picky but this should probably be a `case` - `esac` function

Reply:
Allright done